search/includes/classes/commands/class-corecommand.php

```diff
@@ -121,44 +121,78 @@ private function list_indexes() {
 	}
 
 	protected function maybe_setup_index_version( $assoc_args ) {
```

Comment:
This method looks a bit intense now, can we make it less of a 
![image](https://user-images.githubusercontent.com/459254/215860885-26a08ad9-76c6-4951-8132-f0504ac43fbc.png)